content/blog/pulumi-azure-2-0/_index.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Announcing Pulumi Azure Provider 2.0"
+authors: ["mikhail-shilkov"]
+tags: ["Azure"]
+date: "2020-03-11"
+meta_desc: "Announcing the 2.0 release of Pulumi Azure provider and what's new in it."
+---
+
+We are happy to announce the release of a new major version of Pulumi Azure provider. Pulumi Azure 2.0 is based on the [2.0 release](https://www.terraform.io/docs/providers/azurerm/guides/2.0-upgrade-guide.html) of the upstream provider and brings several improvements and breaking changes.
```

Comment:
Missing the article 'the':
"a new major version of **the** Pulumi Azure provider."